androidtv/constants.py

```diff
@@ -268,18 +268,27 @@
 
 
 # Apps
+APP_AE_TV = 'com.aetn.aetv.watch'
+APP_APPLE_TV = 'com.apple.atve.amazon.appletv'
```

Comment:
Can you please put this in alphabetical order.

Reply:
Oops, thought I did this, my bad